Assignment/ComplexTest.c

```diff
@@ -0,0 +1,56 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Vinitha Perumal - GET
+// ------------------------------------------------------------------
+// ComplexTest.c
+// Program on Test1.2 branch.
+// ------------------------------------------------------------------------------------------------
+#include "Complex.h"
+#include <stdio.h>
+#include <math.h>
+
+void CompareComplex (ComplexNumber actual, ComplexNumber expected, const char* operation);
+void TestCases ();
+
+void CompareComplex (ComplexNumber actual, ComplexNumber expected, const char* operation) {
+   printf ("%s: %s", operation, (actual.Real == expected.Real && actual.Img == expected.Img) ?

```

Comment:
Printing functionality should be kept separate from core (main) functionality. In this case it would be better to have an IsEqual function for Complex numbers (which you can make a part of Complex.c) and print "pass" or "fail" in your test func based on the result from IsEqual()